resources/views/emails/admin/main.blade.php

```diff
@@ -8,37 +8,34 @@
 
 </head>
 
-<body>
+<body style="padding: 1em; background: #e9eaed;">
 
-    <table style="width:100%;">
+<div style="background: #ffffff; padding: 1em;">
 
-        <tbody>
+    <div style="background: #3870a9; border-radius: .26em .26em 0 0; padding: 2.6em 0;">
 
-            <tr>
-                <td>
-                    {{--For Example--}}
-                    {{--<img src="{{ $message->embed(public_path(config('image.admin-email-logo'))) }}"--}}
-                    {{--alt="logo" style="width: 4em;>--}}
-                </td>
-            </tr>
+        <img src="{{ $message->embed(public_path('images/logo/logo-0001.png')) }}"
```

Comment:
Let's not embed please. Just use absolute URL.
Emails will be lighter and logo can be changed remotely.